blueprints/addon/index.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     contents.devDependencies['ember-disable-prototype-extensions'] = '^1.1.0';
 
     // add `ember-try` to addons by default
-    contents.devDependencies['ember-try'] = '~0.0.8';
+    contents.devDependencies['ember-try'] = '^0.1.2';
```

Comment:
Are you sure we want a `^`, since the jump from 0.0 to 0.1 looks like it was a breaking change? 0.1 to 0.2 could also be a breaking change.